bn-models/src/models/_lookups/port.ts

```diff
@@ -6,17 +6,71 @@ export const PortTypeName = 'port';
 // TODO add an at-sea transfer Port ID to DB
 
 export interface Port extends BaseLookup {
-  name?: string;
-  code?: string;
-  group?: string;
-  state?: string;
+  name?: string;  // PORT_NAME from PORTS table
+  state?: string; // STATE from PORTS table
+  pacfinPortCode: string; // PORT_CODE from PORTS table
+  pacfinPortGroupCode: string; // PCGROUP from PORT_DECODER table
+  wcgopPortGroup?: string; // PORT_GROUP from PORTS table
+  ifqPortId?: number; // not sure where this comes from
+  ifqPortCode?: number; // IFQ_PORT_CODE from PORTS table
 
   legacy?: {
-    portId?: number;
-    obsprodLoadDate?: BoatnetDate;
-
-    // Possibly legacy, unclear what these values represent.
-    ifqPortId?: number;
-    ifqPortCode?: number;
+    portId?: number; // PORT_ID from PORTS table
+    obsprodLoadDate?: BoatnetDate; // from PORTS table
   };
 }
+
+
+
+/* New Port
+
+{
+  "_id": "42e32d2f590983b0ec662157ce1f57ba",
+  "_rev": "1-e65da588c46ec95951c6157b0b9cb754",
+  "type": "port",
+  "name": "CHARLESTON (COOS BAY)",
+  "state": "OR",
+  "pacfinPortCode": "COS",
+  "pacfinPortGroupCode": "CBA",
+  "wcgopPortGroup": "CB",
+  "ifqPortId": 262,
+  "ifqPortCode": 34,
+  "createdBy": 1180,
+  "createdDate": "2000-01-03T13:54:59-08:00",
+  "updatedBy": "**** ***** (1331)",
+  "updatedDate": "2019-10-16T08:57:54-07:00",
+  "legacy": {
+    "portId": 11419,
+    "obsprodLoadDate": "Invalid date",
+  }
+
+  ...optionally:
+  isAshop: true
+  isActive: true
+}
+
+*/
+
+/* Old Port
```

Comment:
I like the new port example, but not sure I'd reference the old one. Also, it seems a bit cumbersome to update the old version and new whenever we make a change. If anything you could record the old version in the conversation/comments section in github and post it there, but don't think it's needed in the code. If you remove old port we could also rename new port to current port. Otherwise looks good!

Reply:
yeah, I agree, doesn't seem like an old example is strictly necessary.  In this case, I just copied an existing port doc into the model, as it seemed like a good way to show the proposed changes.